flagsmith/assets/dashboards/flagsmith-dashboard.json

```diff
@@ -0,0 +1,92 @@
+{
+  "title": "Flagsmith Dashboard",
+  "description": null,
```

Comment:
Could you add a short description describing what's available on the dashboard?

Reply:
OK that's added thanks